ASIM/dev/ASimTester/filteringTest/filteringTest.py

```diff
@@ -0,0 +1,191 @@
+import unittest
+import os
+import sys
+import yaml
+import contextlib
+from datetime import datetime, timedelta, timezone
+from azure.monitor.query import LogsQueryClient, LogsQueryStatus
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+from azure.identity import DefaultAzureCredential
+from schemasParameters import all_schemas_parameters
```

Comment:
I think this would not work if the script is run not from the current directory.  For now, might be easiest to just include the dictionary in the primary script.